Directory.Build.props

```diff
@@ -48,4 +48,12 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
 
+  
+  <PropertyGroup>
+    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
+  </ItemGroup>
+
```

Comment:
Just an FYI, our Directory.Build.props are messed up. 
There are 5 in this repo, (1 from the four former repos, and 1 in the root that I created while trying to consolidate the others).
Hopefully, I can fix this next semester.

I'm telling you because changes here may be unpredictable on the build server. I think you're doing the right thing by testing these changes in the csproj. 
For the build server, you should be changing THIS file: https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/Directory.Build.props